core/core/src/scripts/helpers/simple-log-collector.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Copyright 2019 ABB. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# USAGE: ./logs.sh /home/my_user/my_logs
+
+OUTPUT_LOCATION=$1
+
+log_locations=("/var/log/audit/audit.log" "/var/log/auth.log" "/var/log/firewalld" "/var/log/haproxy.log" "/var/log/kafka/server.log" "/var/log/messages" "/var/log/secure" "/var/log/syslog")
+
+mkdir -p "$OUTPUT_LOCATION/$HOSTNAME"
+printf "[CREATED]: Directory %s\n" "$OUTPUT_LOCATION/$HOSTNAME"
+
+for ix in ${!log_locations[*]}
+do
+    
+    if [ -f ${log_locations[$ix]} ]
```

Comment:
I'd suggest using double brackets `[[` (Bash builtin) instead of `[` (external test command)

https://google.github.io/styleguide/shell.xml?showone=Test,_%5B_and_%5B%5B#Test,_%5B_and_%5B%5B